configs/rules.json

```diff
@@ -416,6 +416,10 @@
     "max_days_in_cache": 7,
     "days_lookup": 7
   },
+  "worksforme": {
+    "max_days_in_cache": 7,
+    "days_lookup": 7
```

Comment:
We should look for old bugs that have not changed recently. I suggest looking up bugs created in the last year but not changed in the last month.

Reply:
> We should look for old bugs that have not changed recently. I suggest looking up bugs created in the last year but not changed in the last month.

@suhaibmujahid, I have made updates [here](https://github.com/mozilla/bugbot/pull/2362/commits/664752dfeb637e78fe05b7307a5e6a859b40bda5); I'm not sure if these changes reflect the ones you suggested.